README.md

```diff
@@ -1,2 +1,48 @@
-# medium-guidelines
-Guidelines and good practices for Medium publications
+# Medium Guidelines
+
+This guidelines are made for people who love writing amazing stories on Medium. As developers, we often miss the opportunity to get benefits from sharing our experiences, knowledge and thinking. 
+
+This document intend to be clear, consistent and brief.
```

Comment:
Please, correct with `This document intends`. Also, I'm not sure this should appear in the README. If the document intends to be clear, brief and consistent, readers should be able to see it without us having to specify it. If this was specified for possible contributors, then maybe this is something we should add in a CONTRIBUTING file.